test/iidm/converter/xml/ShuntCompensatorRoundTripTest.cpp

```diff
@@ -111,8 +107,8 @@ BOOST_FIXTURE_TEST_CASE(ShuntLinearRoundTripTest, test::ResourceFixture) {
 BOOST_FIXTURE_TEST_CASE(ShuntNonLinearRoundTripTest, test::ResourceFixture) {
     test::converter::RoundTrip::roundTripVersionedXmlTest("nonLinearShuntRoundTripRef.xml", IidmXmlVersion::CURRENT_IIDM_XML_VERSION());
 
-    // backward compatibility from version 1.2
-    test::converter::RoundTrip::roundTripVersionedXmlFromMinToCurrentVersionTest("nonLinearShuntRoundTripRef.xml", IidmXmlVersion::CURRENT_IIDM_XML_VERSION());
+    // backward compatibility from version 1.3
+    test::converter::RoundTrip::roundTripVersionedXmlFromMinToCurrentVersionTest("nonLinearShuntRoundTripRef.xml", IidmXmlVersion::V1_3());
```

Comment:
Why is it not possible to ensure all version? Is it necessary to do it in two different steps?